bin/citgm-all.js

```diff
@@ -55,7 +73,9 @@ const options = {
   level: app.verbose,
   npmLevel: app.npmLoglevel,
   timeoutLength: app.timeout,
-  tmpDir: app.tmpDir
+  tmpDir: app.tmpDir,
+  includeTags: app.includeTags,
```

Comment:
If you do this I think you'll have to change all the `if(app.includeTags)` conditions to `if(app.includeTags.length)`.